runtime/j9vm/javanextvmi.cpp

```diff
@@ -507,6 +507,18 @@ JVM_VirtualThreadUnmount(JNIEnv* env, jobject vthread, jboolean hide, jboolean l
 	}
 }
 
+JNIEXPORT void JNICALL
+JVM_VirtualThreadStart(JNIEnv* env, jobject vthread)
```

Comment:
Space before, not after `*`; here and for `JVM_VirtualThreadEnd`.

Reply:
Adjusted